### PR TITLE
Hooli style popup

### PR DESCRIPTION
**Pull request by drewzag**
### Hooli-style Popup

[Demo](https://drewzag.github.io/hooli-style-popup/) |
[Code base](https://github.com/drewzag/hooli-style-popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

File: submissions/drewzag/hooli-style-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+            <a href="#" class="header__link">Mail</a>
+          </li>
+          <li class="header__item">
+            <a href="#" class="header__link">Images</a>
+          </li>
+          <li class="header__item">
+            <input type="checkbox" class="header__checkbox" id="checkbox" />
+            <label for="checkbox">
+              <img src="/popup-icons/popup-button.png" alt="" class="popup" />
+            </label>
```
Comment (A-Ostrovnyy) [inline submissions/drewzag/hooli-style-popup/index.html:28]: Please fix styles on focus state: [watch video](https://share.vidyard.com/watch/zy6cyn6ShCLu43EeFyo2Qc?)

File: submissions/drewzag/hooli-style-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+            <div class="popup__wrapper">
+              <ul class="popup__list">
+                <li class="popup__item">
+                  <a href="#" class="popup__item-link">
+                    <img src="/popup-icons/calendar.png" alt="" class="popup-image" />
```
Comment (A-Ostrovnyy) [inline submissions/drewzag/hooli-style-popup/index.html:34]: Read how and when to populate alt attribute value [Как правильно написать alt-текст](https://htmlacademy.ru/blog/articles/alt-text)

File: submissions/drewzag/hooli-style-popup/index.html [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+                    <img src="/popup-icons/documents.png" alt="" class="popup-image" />
+                    <span class="popup-text">Document</span>
+                  </a>
+                </li>
+              </ul>
+
+              <input type="checkbox" class="more__checkbox" id="checkbox__more-less" />
+              <label for="checkbox__more-less">
+                <span class="show-more">Show More</span>
+              </label>
```
Comment (A-Ostrovnyy) [inline submissions/drewzag/hooli-style-popup/index.html:109]: Please fix styles on the focus state to indicate the current position of focus on the page. Watch provided  [video](https://share.vidyard.com/watch/zy6cyn6ShCLu43EeFyo2Qc?)

File: submissions/drewzag/hooli-style-popup/style.css [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+  height: 1.5em;
+}
+
+.header {
+  background-color: bisque;
+  padding: 10px 100px 0px 25px;
+  height: 3em;
+}
+
+.nav {
```
Comment (A-Ostrovnyy) [inline submissions/drewzag/hooli-style-popup/style.css:48]: Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
Sorry for the late response.

File: submissions/drewzag/hooli-style-popup/index.html [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+          </li>
+          <li class="header__item">
+            <a href="#" class="header__link">Images</a>
+          </li>
+          <li class="header__item">
+            <input type="checkbox" class="header__checkbox" id="checkbox" />
+            <label for="checkbox" class="label__checkbox">
+              <img
+                src="./popup-icons/popup-button.png"
+                alt="open or close popup menu"
```
Comment (A-Ostrovnyy) [inline submissions/drewzag/hooli-style-popup/index.html:29]: DEcorative images no need populated `alt` attribute, icon is a decorative image.)

File: submissions/drewzag/hooli-style-popup/index.html [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+                class="popup"
+              />
+            </label>
+
+            <div class="popup__wrapper">
```
Comment (A-Ostrovnyy) [inline submissions/drewzag/hooli-style-popup/index.html:34]: Please fix mobile version 
image

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good progress!

**Comment by drewzag** [comment]
@A-Ostrovnyy I hope, I did it wright this time)
Thank you for reviewing my code

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**